examples/nextjs-with-typescript/pages/MuxVideo.tsx

```diff
@@ -1,14 +1,18 @@
 import Link from "next/link";
-import { useRef, useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import MuxVideo from "@mux-elements/mux-video-react";
 
 const INITIAL_AUTOPLAY = false;
 const INITIAL_MUTED = false;
 
 function MuxVideoPage() {
   const mediaElRef = useRef(null);
-  const [autoplay, setAutoplay] = useState(INITIAL_AUTOPLAY);
+  const [autoplay, setAutoplay] = useState<"muted" | boolean>(INITIAL_AUTOPLAY);
   const [muted, setMuted] = useState(INITIAL_MUTED);
+  const [paused, setPaused] = useState<boolean | undefined>(true);
+  useEffect(() => {
+    if (!mediaElRef.current) return;
+  }, [paused]);
```

Comment:
Does this have some kind of effect or should it be removed?

Reply:
lol yeah should be removed. I was stubbing paused toggle but then I realized that `MuxVideo`/`mux-video` doesn't have `paused`.